argocd/apps/values.yaml

```diff
@@ -29,3 +29,5 @@ apps:
     enabled: true
   vault:
     enabled: true
+  minio:
+    enabled: true
```

Comment:
Could you please set this to `false` as default and set it to `true` in `k3s-docker` module?